### ISSUE TITLE
System Error when running Ganache 2.1.2 on win32

### ISSUE THREAD

**Issue by eliudrojas** [trufflesuite/ganache#2056]
PLATFORM: win32
GANACHE VERSION: 2.1.2

EXCEPTION:
```
Error: Hardfork muirGlacier not set as supported in supportedHardforks
    at Common.setHardfork (D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\node_modules\ethereumjs-common\src\index.ts:112:13)
    at new Common (D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\node_modules\ethereumjs-common\src\index.ts:79:12)
    at new VM (D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\node_modules\ethereumjs-vm\dist\index.js:62:21)
    at BlockchainDouble.createVMFromStateTrie (D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\lib\blockchain_double.js:130:14)
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\lib\blockchain_double.js:85:36
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\lib\blockchain_double.js:191:5
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\lib\database\leveluparrayadapter.js:129:14
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\lib\database\leveluparrayadapter.js:24:16
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\node_modules\level-sublevel\shell.js:101:15
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\node_modules\level-sublevel\nut.js:121:19
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\node_modules\encoding-down\index.js:51:21
    at D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\node_modules\cachedown\index.js:58:21
    at ReadFileContext.callback (D:\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\ganache-core\lib\database\filedown.js:26:14)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:420:13)
```

**Comment by davidmurdoch** [comment]
Fixed in v2.4.0. Please update https://www.trufflesuite.com/ganache :-)

**Comment by eliudrojas** [comment]
Thx
